packages/react-mutation-mapper/src/component/column/Annotation.tsx

```diff
@@ -219,51 +215,68 @@ export function sortValue(annotation: IAnnotation): number[] {
     return _.flatten([
         oncoKbSortValue(annotation.oncoKbIndicator),
         civicSortValue(annotation.civicEntry),
-        // MyCancerGenome.sortValue(annotationData.myCancerGenomeLinks),
+        myCancerGenomeSortValue(annotation.myCancerGenomeLinks),
         hotspotSortValue(annotation.isHotspot, annotation.is3dHotspot),
         annotation.isOncoKbCancerGene ? 1 : 0,
     ]);
 }
 
+export function renderAnnotation(
+    annotation: IAnnotation,
+    enableOncoKb: boolean,
+    enableCivic: boolean,
+    enableMyCancerGenome: boolean,
+    enableHotspot: boolean,
+    pubMedCache?: MobxCache,
+    userEmailAddress?: string
+): JSX.Element {
+    return (
```

Comment:
whats the point of pulling this out?

Reply:
I wasn't able to directly use the `Annotation` component for the PatientView mutation table (because it doesn't use Mutation data), so instead calling this function to render it there: https://github.com/cBioPortal/cbioportal-frontend/pull/3119/files#diff-a4d36866c4ebe7d81ddb47337d37759eR273. Ideally `Annotation` component should be more generic, but I am leaving that for another PR.

Reply:
ah okay. I didn't realize you were using it somewhere else. This is fine but we've starting using react functional components for stuff like this, maybe it would be good to do that here, call it `GenericAnnotation` or something